package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fuzzy-finder",
-  "version": "0.70.0",
+  "version": "0.72.0",
```

Comment:
Not sure what happened here, but it looks like this is an old version of package.json - can you try merging / rebasing master again? The version should be 0.81.0